### ISSUE TITLE
No Tatin commands after Installation

### ISSUE THREAD

**Issue by kdtp** [aplteam/Tatin#146]
I used this installation instructions: https://tatin.dev/Assets/docs/InstallingAndUpdatingTheTatinClient.html

I installed it for Windows and Macos using Dyalog 18.2 64 Bit Unicode.

After restarting there are no tatin commands ]Tatin.*:

<img width="226" alt="image" src="https://github.com/aplteam/Tatin/assets/78899531/7fd52dce-b157-4a55-bc57-f11be2dcce1f">

![image](https://github.com/aplteam/Tatin/assets/78899531/864bdc43-8b9c-485c-8a5d-b0e97b64882d)

But I found it in ⎕SE.Tatin.Tatin  in the explorer:

![image](https://github.com/aplteam/Tatin/assets/78899531/2e193176-01ab-43fc-b5bc-8cd4a02d4bba)

<img width="162" alt="image" src="https://github.com/aplteam/Tatin/assets/78899531/f7097b8e-3b9a-4c3a-8b41-aeeccbf07e00">

**Comment by aplteam** [comment]
There is a flaw in the process. I need to reconsider.

As a workaround please execute this:

Windows:
```
]SALT.Settings cmddir ",C:\Users\kai\Documents\Dyalog APL-64 18.2 Unicode Files\StartupSession"
```

Mac OS:
```
]SALT.Settings cmddir ",/Users/<⎕AN>/dyalog.182U64.files/StartupSession"
```

_**Attention:**_ notice the leading `,` which means "add" rather than "replace".

Only then will Dyalog scan the directory for user command scripts.

I will soon either publish an updated version of the documentation that includes the above statements or publish a new version of Cider & Tatin that corrects the underlying problem.

**Comment by aplteam** [comment]
Fixed with version 0.102.0
